dist/tools/tapsetup/tapsetup

```diff
@@ -174,12 +174,9 @@ while true ; do
     esac
 done
 
-if [ -z "${USER}" ]; then
-    export USER=$(id -un)
-    if [ -z "${USER}" ]; then
-        echo 'need to export $USER'
-        exit 1
-    fi
+if [ -z "${SUDO_USER}" ]; then
+    echo 'need to export $SUDO_USER'
```

Comment:
It would be more informative to say that the script needs to be run with sudo. The message could be confusing as it is now.

Reply:
Fixed. However I still prefer to name the symptom first, then provide a cure ;-).